### PR TITLE
Fix: Broken Links and Added MAMP instructions

### PR DESCRIPTION
**Pull request by punwai**
•	Broken links in INSTALL.md is fixed.
•	A section on installation using MAMP on mac is added.
•	Screenshots added for MAMP

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -392,6 +392,104 @@ Launch XAMPP Control Panel and navigate to LibreEHR Setup in your browser: `\Lib
 
  **Once you see the screen above, you have successfully setup LibreEHR! Congratulations!**
 
+## On Mac With MAMP (For Local Testing)
+Apache can be used on Mac to install LibreHealth. However, if you have installation problems, you can use MAMP.
+
+#### Setup
+Install MAMP version 3.5.6 or under (Newer versions come with PHP 7.1 which is not compatible with LibreHealthEHR. If youdo install a newer version, make sure to use PHP versions under 7.1). You can find older versions.
+
+#### Note: PHP must not be version 7.1 or above. Version 7.0 and 5.6 can both be used.
```
Comment (robbyoconnor) [inline INSTALL.md]: Please state that Version 7.0 and 5.6 are close to EOL -- please discourage.

File: INSTALL.md [hunk 2]
```diff
@@ -392,6 +392,104 @@ Launch XAMPP Control Panel and navigate to LibreEHR Setup in your browser: `\Lib
...
+
+ Now navigate to Preferences > Ports and change your Apache Port to 80 and MySQL port to 3306. This can be done by pressing 'Set Web and MySQL ports to 80 & 3306'.
+
+ ![MAMPSettings](./Documentation/1_Installing/images/MAMP_installation/Settings.png)
+
+ Finally, you should make sure that you have got the right PHP version.
+
+ ![MAMPPHP](./Documentation/1_Installing/images/MAMP_installation/PHP.png)
+
+ Version 5.6.10 is recommended but any version which is not 7.1 and newer is fine.
```
Comment (robbyoconnor) [inline INSTALL.md]: No. This is EOL. Discourage it. PHP 7.1 or later. See http://php.net/supported-versions.php

File: INSTALL.md (hunk 1 above)
Comment (robbyoconnor) [reply INSTALL.md]: Don't even mention them.

File: INSTALL.md [hunk 3]
```diff
@@ -392,6 +392,104 @@ Launch XAMPP Control Panel and navigate to LibreEHR Setup in your browser: `\Lib
 
  **Once you see the screen above, you have successfully setup LibreEHR! Congratulations!**
 
+## On Mac With MAMP (For Local Testing)
+Apache can be used on Mac to install LibreHealth. However, if you have installation problems, you can use MAMP.
+
+#### Setup
+Install MAMP version 3.5.6 or under (Newer versions come with PHP 7.1 which is not compatible with LibreHealthEHR. If youdo install a newer version, make sure to use PHP versions under 7.1). You can find older versions.
```
Comment (robbyoconnor) [inline INSTALL.md]: Check this closely

File: INSTALL.md (hunk 1 above)
Comment (robbyoconnor) [inline INSTALL.md]: http://php.net/supported-versions.php

Both PHP 5.6.x 7.0.x are close to the end of the line.

File: INSTALL.md [hunk 4]
```diff
@@ -392,6 +392,104 @@ Launch XAMPP Control Panel and navigate to LibreEHR Setup in your browser: `\Lib
...
+
+#### Note: PHP must not be version 7.1 or above. Version 7.0 and 5.6 can both be used.
+
+Firstly, download [MAMP](https://mamp.info/en/) version 3.5.6. Currently, we do not support PHP versions 7.1. MAMP 3.5.6 comes with PHP 7.0 so you can use it without much configuration. Older Versions (including version 3.5.6) can be found [here](https://mamp.info/en/downloads/older-versions/)
+
```
Comment (robbyoconnor) [inline INSTALL.md]: Let's ensure that people use a proper version -- which is not close to EOL -- ensure that _PHP 7.1 or better is used.

File: INSTALL.md [hunk 5]
```diff
@@ -392,6 +392,104 @@ Launch XAMPP Control Panel and navigate to LibreEHR Setup in your browser: `\Lib
...
+ ![MAMPPHP](./Documentation/1_Installing/images/MAMP_installation/PHP.png)
+
+ Version 5.6.10 is recommended but any version which is not 7.1 and newer is fine.
+
+2. To make sure the LibreHealthEHR functions properly, you will need to make some changes to your php.ini file. Different versions of php have different php.ini files.
+For PHP 7.0.8
+`/Applications/MAMP/conf/php7.0.8/php.ini`
+For PHP 5.6.10
+`/Applications/MAMP/conf/php5.6.10/php.ini`
+
```
Comment (robbyoconnor) [inline INSTALL.md]: Ensure that you are not advocating for people to use a version that will be deprecated and unsupported in a month and change.

File: INSTALL.md (hunk 4 above)
Comment (muarachmann) [reply INSTALL.md]: sure @punwai please see comment thanks @robbyoconnor

Comment (punwai) [reply INSTALL.md]: @robbyoconnor  @muarachmann Is there support for 7.1 ++ yet? Or is it going to be implemented soon

**Comment by punwai** [comment]
@robbyoconnor @muarachmann Just made changes so that the tutorial now uses MAMP 5.1 with php 7.2.x and 7.1.x

**Comment by punwai** [comment]
Also for some reason the file I submitted earlier was not the version I was supposed to submit (I probably forgot to push it or something). But it is all updated now.

**Comment by tmccormi** [comment]
Is this a GCI task?

**Comment by robbyoconnor** [comment]
Yes.

**Comment by robbyoconnor** [comment]
It's okay if it takes time to approve this -- there's no rush to merge this @tmccormi --- the student's task has already been approved but if there are changes you see that need to be made, they should fix it.

File: INSTALL.md (hunk 5 above)
Comment (aethelwulffe) [reply INSTALL.md]: Yeah, actually no 7.0 or earlier should be used.  There are issues.  Might as well drop them from your instructions, or at least note the "minimum version" and "configurator beware"

**Pull request merged**